packages/api/pw-test.config.cjs

```diff
@@ -56,8 +56,7 @@ module.exports = {
       stdout.toString().includes('Server started on: http://localhost:9094')
     ) {
       console.log('⚡️ Mock cluster started.')
-      await execa(cli, ['db', '--start', '--project', project])
-      await delay(2000)
+      await execa(cli, ['db', '--init', '--start', '--project', project])
```

Comment:
Added `--init` here - not sure how things were working in CI before...

Reply:
start seems to be enough run everything need, did it fail for you locally ?

Reply:
start seems to run everything need, did it fail for you locally ?

Reply:
Ah seems to work now if I remove it so I'll revert.